File: app/src/main/java/com/example/taskweek2/model/LoginResponse.java
package com.example.taskweek2.model;

public class LoginResponse {
	private LoginData data;
	private String message;
	private boolean status;
	private String token;

	public LoginData getData(){
		return data;
	}

	public String getMessage(){
		return message;
	}

	public boolean isStatus(){
		return status;
	}

	public String getToken(){
		return token;
	}
}

File: app/src/main/java/com/example/miniproject/model/LoginResponse.kt
package com.example.miniproject.model
import androidx.room.Entity
import com.google.gson.annotations.SerializedName

@Entity(tableName = "Login")
data class LoginResponse(
    var query: String? = null,
    @field:SerializedName("data") val data :LoginData,
    @field:SerializedName("message") val message :String,
    @field:SerializedName("status") val status :String,
    @field:SerializedName("token") val token :String,

)